playground/vue3/src/pages/avatar.vue

```diff
@@ -4,10 +4,9 @@ import { OkuAvatar, OkuAvatarFallback, OkuAvatarImage } from '@oku-ui/avatar'
 
 <template>
   <div class="mt-4">
-    <OkuAvatar class="w-40 h-40 bg-gray-400 items-center inline-block overflow-hidden rounded-full">
+    <OkuAvatar src="https://images.unsplash.com/photo-1492633423870-43d1cd2775eb?&w=128&h=128&dpr=2&q=80" class="w-40 h-40 bg-gray-400 items-center inline-block overflow-hidden rounded-full">
```

Comment:
src location is wrong, `OkuAvatarImage` only with used src.

Reply:
please check radix avatar -> https://github.com/radix-ui/primitives/blob/main/packages/react/avatar/src/Avatar.tsx#L64

Reply:
The image tag has src, please pay attention to the html tags.the avatar is in a span position, the src doesn't go from there, usage mistake